Formula/hugo.rb

```diff
@@ -16,7 +16,9 @@ class Hugo < Formula
   depends_on "go" => :build
 
   def install
+    ENV["GOPATH"] = "#{HOMEBREW_CACHE}/go_cache"
```

Comment:
Do we want to follow the style used by `glide` formulae?

```
ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
```

Reply:
Aye, that's probably a good step 👍.